bigtable/client/instance_admin.cc

```diff
@@ -107,6 +107,77 @@ google::bigtable::admin::v2::Instance InstanceAdmin::CreateInstanceImpl(
   return result;
 }
 
+std::future<google::bigtable::admin::v2::Instance>
+InstanceAdmin::UpdateInstance(google::bigtable::admin::v2::Instance* instance,
+                              google::protobuf::FieldMask* update_mask) {
+  return std::async(std::launch::async, &InstanceAdmin::UpdateInstanceImpl,
+                    this, instance, update_mask);
+}
+
+google::bigtable::admin::v2::Instance InstanceAdmin::UpdateInstanceImpl(
+    google::bigtable::admin::v2::Instance* instance,
```

Comment:
Passing parameters as pointers is not a great idea, why do we need to do this here?

Reply:
changed to `InstanceUpdateconfig`